### DISCUSSION TITLE
Misterious log entry 😄

### DISCUSSION THREAD

**Discussion by skeyby** [apache/couchdb#2965] [Q&A] [answered]
```
[error] 2020-06-25T07:52:37.644079Z couchdb@10.133.136.126 <0.3826.204> bd5c2120fc req_err(3717872069) unknown_error : normal
    [<<"chttpd:catch_error/3 L356">>,<<"chttpd:handle_req_after_auth/2 L322">>,<<"chttpd:process_request/1 L303">>,<<"chttpd:handle_request_int/1 L243">>,<<"mochiweb_http:headers/6 L128">>,<<"proc_lib:init_p_do_apply/3 L249">>]
[error] 2020-06-25T07:53:14.061720Z couchdb@10.133.136.126 <0.19345.203> 8a1fabea0b req_err(3717872069) unknown_error : normal
    [<<"chttpd:catch_error/3 L356">>,<<"chttpd:handle_req_after_auth/2 L322">>,<<"chttpd:process_request/1 L303">>,<<"chttpd:handle_request_int/1 L243">>,<<"mochiweb_http:headers/6 L128">>,<<"proc_lib:init_p_do_apply/3 L249">>]
[error] 2020-06-25T07:54:16.360572Z couchdb@10.133.136.126 <0.24324.200> bb2251ef97 req_err(3717872069) unknown_error : normal
    [<<"chttpd:catch_error/3 L356">>,<<"chttpd:handle_req_after_auth/2 L322">>,<<"chttpd:process_request/1 L303">>,<<"chttpd:handle_request_int/1 L243">>,<<"mochiweb_http:headers/6 L128">>,<<"proc_lib:init_p_do_apply/3 L249">>]
[error] 2020-06-25T07:54:16.360577Z couchdb@10.133.136.126 <0.11982.202> 6e161734ae req_err(3717872069) unknown_error : normal
    [<<"chttpd:catch_error/3 L356">>,<<"chttpd:handle_req_after_auth/2 L322">>,<<"chttpd:process_request/1 L303">>,<<"chttpd:handle_request_int/1 L243">>,<<"mochiweb_http:headers/6 L128">>,<<"proc_lib:init_p_do_apply/3 L249">>]
```

I have thousands of those in my couchdb2.log.

Is it something to worry about? "Unknown_error: normal".... mmmm

CouchDB 2.3.1 on FreeBSD 12.1

Thanks

**Comment by rjharmon** [comment, marked as answer]
This happens for me when a client process disconnects from an open _changes feed after 10s (or is it 10s of inactivity?).  The "normal" is supposed to reassure you, but the ambiguity is almost as disturbing as the "unknown" part.

Maintainers, can this condition be detected more specifically so that the normal case can become a simple "client disconnect" message - or even lowered to [warn] or [info] severity?

**Comment by wohali** [comment]
We are tracking the actual bug report on this in #1600.

It does appear to just be a sporadic warning thrown when the client disconnects while CouchDB is waiting for a command, and can safely be ignored.

So far no one has reported this with CouchDB 3.x; try upgrading?

**Comment by skeyby** [comment]
@wohali Hi Joan, I'd like to, but Couch 3.x isn't available for FreeBSD yet.

I'm in contact with Dave Cottlehuber who's working on that (https://reviews.freebsd.org/D24615).
